apps/src/code-studio/pd/teachercon1819_registration/Joining.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {
+  FormGroup,
+  ControlLabel
+} from 'react-bootstrap';
+import Teachercon1819FormComponent from './Teachercon1819FormComponent';
+import UsPhoneNumberInput from "../form_components/UsPhoneNumberInput";
+
+import { TeacherSeatAcceptanceOptions } from '@cdo/apps/generated/pd/teachercon1819RegistrationConstants';
+
+export default class Joining extends Teachercon1819FormComponent {
+  static associatedFields = [
+    'preferredFirstName',
+    'lastName',
+    'email',
+    'phone',
+    'teacherAcceptSeat',
+    'teacherWaitlistExplain',
+    'teacherDeclineExplain',
+  ]
```

Comment:
This could be `Object.keys(Joining.labels)`, if moved below the `labels` definition.